_schema.graphql

```diff
@@ -44,6 +47,11 @@ type CollectionCategory {
   collections: [Collection!]!
 }
 
+type CollectionGroup {
+  name: String!
```

Comment:
could we add a `type` property which would be `artist_series`, `featured` or `other`?
reasoning is, if we decide to change the ordering of the rails in the UI, we can’t rely on the name of the group as its customizable